src/custom/components/CurrencyInputPanel/index.tsx

```diff
@@ -14,12 +14,20 @@ import CurrencyInputPanelMod, {
   Container,
   StyledBalanceMax,
 } from './CurrencyInputPanelMod'
+import CurrencySearchModalUni from '@src/components/SearchModal/CurrencySearchModal'
 import { RowBetween } from 'components/Row'
 import { FeeInformationTooltipWrapper } from 'components/swap/FeeInformationTooltip'
 
 import { StyledLogo } from 'components/CurrencyLogo'
 import { LONG_LOAD_THRESHOLD } from 'constants/index'
 
+export const CurrencySearchModal = styled(CurrencySearchModalUni)`
```

Comment:
Moved the custom code from the MOD file into this index file.

Reply:
👍